sources/src/shortlived-token/shortlived-token.ts

```diff
@@ -0,0 +1,164 @@
+import * as httpm from 'typed-rest-client/HttpClient'
```

Comment:
The "package" for this file should not be "shortlived-token", but something more general purpose. Let's use "develocity".

Reply:
Also, for consistency we should name this `short-lived-token`.